submissions/NatalyaBortsova/popup/index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Popup</title>
+  <link rel="stylesheet" href="css/style.min.css">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="header">
+    <div class="container">
+      <nav class="header__menu menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a class="menu__link animation" href="#" tabindex="1">
+              Mail
+            </a>
+          </li>
+          <li class="menu__item">
+            <a class="menu__link animation" href="#" tabindex="2">
+              Images
+            </a>
+          </li>
+          <li class="menu__item">
+            <input id="checkbox" class="menu__item-checkbox" type="checkbox">
+            <span class="menu__item-check" tabindex="3"></span> 
+            <label for="checkbox" class="menu__item-label" tabindex="-1">
```

Comment:
Attribute `tabindex` is redundant here you should highlight `label` when `input` is in focus.